kitsune/questions/views.py

```diff
@@ -604,6 +605,7 @@ def aaq(
             statsd.incr("questions.aaq.select-product")
 
     if request.method == "GET":
+        featured = []
```

Comment:
Instead of adding an empty list, you could tweak the `get_featured_articles` method to return an empty list if there aren't any articles